fitbenchmarking/core/fitting_benchmarking.py

```diff
@@ -547,18 +549,20 @@ def perform_fit(controller, options, grabbed_output):
     :type options: fitbenchmarking.utils.options.Options
     :param grabbed_output: Object that removes third part output from console
     :type grabbed_output: fitbenchmarking.utils.output_grabber.OutputGrabber
-    :return: The chi squared and runtime of the fit.
-    :rtype: tuple(float, float)
+    :return: The chi squared, runtime, and emissions of the fit.
+    :rtype: tuple(float, float, float)
     """
     num_runs = options.num_runs
     try:
         with grabbed_output:
             controller.validate()
             # Calls timeit repeat with repeat = num_runs and number = 1
-            runtime_list = timeit.Timer(
-                setup=controller.prepare,
-                stmt=controller.execute
-            ).repeat(num_runs, 1)
+            with EmissionsTracker() as tracker:
+                runtime_list = timeit.Timer(
+                    setup=controller.prepare,
+                    stmt=controller.execute
+                ).repeat(num_runs, 1)
+            emissions = tracker.final_emissions / num_runs
```

Comment:
Do we want to track the prepare call? Currently this is left out of timings.

Reply:
This is a good point - I had set up the emissions tracking this way to avoid having to initialize a new tracker for each run (as each `EmissionsTracker` object can only be started and stopped once), but tracking the prepare call is not ideal - I'll have a look at whether there's a better solution.

Reply:
I've made some changes so that a new `EmissionsTracker` is created for each run - but am finding this is causing Fitbenchmarking to run very slowly - on my computer it takes ~5 seconds to create an `EmissionsTracker` object so it really is painfully slow... will continue to think if there's a way to measure only the call to `controller.fit` without instantiating multiple tracking objects